extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1686,7 +1700,10 @@ export class AmpA4A extends AMP.BaseElement {
           `${this.safeframeVersion};${creative.length};${creative}` +
           `${contextMetadata}`;
       }
-      return this.iframeRenderHelper_(dict({'src': srcPath, 'name': name}));
+      return this.iframeRenderHelper_(
+        dict({'src': srcPath, 'name': name}),
+        false
```

Comment:
I'm a bit confused why we need to set the `letCreativeTriggerRenderStart` value specifically to `false` here?